test/dynamic_config_test.go

```diff
@@ -263,3 +516,193 @@ func getPodIDs(ctx context.Context, aeroCluster *asdbv1.AerospikeCluster) (map[s
 
 	return pidMap, nil
 }
+
+func updateValue(val interface{}) interface{} {
```

Comment:
I think 
For bool type value we can flip it to the opposite for testing dynamic update. 
For string, we can try to update same value